examples/tutorials/hello-gltf/app.ts

```diff
@@ -28,9 +28,9 @@ export default class AppAnimationLoopTemplate extends AnimationLoopTemplate {
     this.loadGLTF('Avocado');
     const modelSelector = document.getElementById('model-select');
 
-    modelSelector.addEventListener('change', e => {
-      this.loadGLTF((e.target as HTMLSelectElement).value);
-    });
+    // modelSelector.addEventListener('change', e => {
```

Comment:
Cut or fix?

Reply:
I just pushed a change to create this UI in code, not the most scalable approach though... do you have any thoughts on how we could add UI?